tests/phpunit/tests/test-media.php

```diff
@@ -79,27 +79,35 @@ function test_attachment_fields_to_edit() {
 		$this->assertFalse( isset( $fields['language'] ) );
 	}
 
+	/**
+	 * @since 3.1 Since the language and translations are updated through a previous AJAX call, we'd rather not perform an unnecessary update now.
```

Comment:
Maybe comment a little bit more inside the test how it works.
Mock of the hook `set_object_terms` triggers by the` edit_post()` function call and then test that there is no language term modification.